### PR TITLE
Node perf tester

### PR DESCRIPTION
**Pull request by TimvdLippe**
Add a node interface for the perf-tester. It uses chrome headless if no port is specified. It parses a series of options which it will navigate to, perform the timings and then extract `window.perfTiming` from the `Runtime`.

### CODE REVIEW

File: options.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[
+    ["original", "http://localhost:8081/youtube/"],
```
Comment (sorvell) [inline options.json]: These should be removed, right?

File: perf.js [hunk 2]
```diff
@@ -1,57 +1,59 @@
...
-    if (console.time) {
-      console.timeEnd('perf');
-    }
-    document.title = time.toFixed(1) + 'ms: ' + document.title;
-    if (window.top !== window) {
-      window.top.postMessage({time: time + 'ms', info: info}, '*');
-    }
-  };
-
+// x-browser compat.
```
Comment (sorvell) [inline perf.js]: The diff here is weird. What happened?

**Review by sorvell — CHANGES_REQUESTED** [review]

File: perf.js (hunk 2 above)
Comment (TimvdLippe) [reply perf.js]: The line endings were mixed. I changed all lines to LF.

**Comment by TimvdLippe** [comment]
@sorvell I updated the line endings, so the diff is less nice :cry: I updated `console._perfEnd` to take an options object to disable the force layout, as well as expose the actual timing in `window.perfTiming`, such that the node script can use it. I also added some documentation on how to use the script.

**Comment by hastebrot** [comment]
>I updated the line endings, so the diff is less nice

Using `w=1` (ignore whitespace; similar to `git diff --ignore-all-space`) in the URL for the diff viewer (compare) helps a bit:
https://github.com/PolymerLabs/perf-tester/pull/4/files?diff=unified&w=1

**Review by sorvell — APPROVED** [review]

**Pull request merged**